nuxt-app/pages/party/create.vue

```diff
@@ -1,156 +1,206 @@
 <script setup lang="ts">
+  import VueDatePicker from '@vuepic/vue-datepicker'
+  import '@vuepic/vue-datepicker/dist/main.css'
+  import SpotButton from '~/components/spot-button.vue'
+
   // Get tRPC client
   const { $client } = useNuxtApp()
+  // Get router
+  const router = useRouter()
 
-  // Keep track of party name input and created party
-  const partyName = ref('')
-  const partyRef = ref<{ name: string; imageURL: string } | undefined>(undefined)
-
-  // Execute tRPC querys
-  const user = $client.auth.getUser.useQuery()
-  const userParties = $client.party.getUserParties.useQuery()
-
-  // Convert blob to base64 DataURL
-  // Reference: https://stackoverflow.com/a/18650249/14906871
-  const blobToDataURL = (blob: Blob): Promise<string> => {
-    return new Promise((resolve) => {
-      const reader = new FileReader()
-      reader.onloadend = () => resolve(reader.result as string)
-      reader.readAsDataURL(blob)
-    })
-  }
-
-  const randomTestImage = async () => {
-    // Get random image blob from unsplash
-    const randomURL = 'https://source.unsplash.com/random/300x300/?party'
-    const res = await fetch(randomURL)
-    const blob = await res.blob()
-
-    // Convert blob to DataURL
-    const dataURL = await blobToDataURL(blob)
-
-    // Split DataURL into data and mime-type
-    const data = dataURL.split(',')[1]
-    const mimeType = dataURL.split(',')[0]?.split(':')[1]?.split(';')[0]
+  // Temp image-file storage
+  const file = ref<File[]>([])
 
-    if (!data || !mimeType) throw new Error('Could not create random image')
-
-    return { data, mimeType }
+  // Keep track of party name input and created party
+  const party = {
+    name: ref(''),
+    description: ref(''),
+    startAutomatically: ref(new Date()),
   }
-
-  const testPartyProcedures = async (partyName: string, del: boolean) => {
-    // Generate random image
-    const { data: imageData, mimeType: imageMimeType } = await randomTestImage()
-
-    // Create new party
-    const party = await $client.party.createParty.mutate({
-      party: {
-        name: partyName,
-        description: 'Test party',
-      },
-      // Add random image to party
-      image:
-        imageData && imageMimeType
-          ? {
-              base64Blob: imageData,
-              mimeType: imageMimeType as 'image/jpeg',
+  const base64Blob = ref<string | undefined>(undefined)
+  const mimeType = ref<'image/jpeg' | 'image/jpg' | 'image/png' | undefined>(undefined)
```

Comment:
Idealerweise Image-Type in `types/trpc.ts` von tRPC inferieren und hier wiederverwenden, damit Frontend und Backend in-Sync bleiben